File: problems/template3.js
// template file
const tc = require("../testcase-3-longest-substring-without-repeating-characters.json")

// insert Solution class here


let isOk = true
const arrInput = tc.input
const arrExpected = tc.expected
for (let i = 0; i < arrInput.length; i++) {
    const solution = lengthOfLongestSubstring(arrInput[i])
    if (solution != arrExpected[i]) {
        isOk = false
        break
    }

}

if (isOk) {
    const output = {
        "result": "OK"
    }
    const data = JSON.stringify(output)
    console.log(data)
}

console.log("test completed")

File: problems/template3.py
# template file
import json
from typing import List


# insert Solution class here



with open("../testcase-3-longest-substring-without-repeating-characters.json", "r") as read_file:
    data = json.load(read_file)

sol = Solution()
isOk = True

listInput = data["input"]
listExpected = data["expected"]

for i, element in enumerate(listInput):
    solution = sol.lengthOfLongestSubstring(element)
    if solution != listExpected[i]:
        isOk = False
        break

if isOk:
    output = {
        "result": "OK"
    }
    json_object = json.dumps(output, indent=4)
    print(json_object)

print("test completed")